test/e2e/clusterinfo/openshift.go

```diff
@@ -93,3 +93,12 @@ func (o *OpenShift) GetInfrastructure() (*config.Infrastructure, error) {
 	}
 	return infra, nil
 }
+
+// IsProxyEnabled queries the Proxy resource to see if a cluster-wide proxy is enabled in this environment
+func (o *OpenShift) IsProxyEnabled() (bool, error) {
```

Comment:
nit: consider renaming this to `ProxyEnabled`, just personal preference as it seems more natural to me:
```
if o.ProxyEnabled{}
vs
if o.IsProxyEnabled{}
```